starter-code/src/components/Form.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+
+class Form extends Component {
+
+  state = {
+    name: '',
+    calories: 0,
+    image: ''
+  };
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name] : event.target.value })
+  }
+
+  handleSubmit = (e) => {
+    // como el formulario, tiene el evento por defecto que es hacer un post, tenemos que meter un 
+    e.preventDefault();
+    this.props.onSubmitForm({name: this.state.name, calories: this.state.calories, image: this.state.image})
```

Comment:
Si haces `lift state up` con `this.props.onSubmitForm`, por qué no hacer lo mismo con todo el estado y así este componente pasa a ser Stateless?. Siempre es mejor desacoplar en la medida de lo posible la lógica de la presentación